spring-cloud-gcp-examples/spring-cloud-gcp-integration-pubsub-example/spring-cloud-gcp-integration-pubsub-example-sender/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+server.port=8081
```

Comment:
This is stand along sample - I would prefer to let end-users to get a gain from using just default 8080 port.
Therefore no this `application.properties` at all!